packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/nodejs/summaryofsteps.md

```diff
@@ -2,6 +2,6 @@ The following diagram shows the sequence of steps for the password recovery use
 
 <div class="common-image-format">
 
-![Displays the password recovery sequence diagram](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-nodejs.png)
+![Sequence diagram displaying the password recovery use case](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-nodejs.png)
```

Comment:
This is a great sequence diagram. If only there is a way to embiggen it when on the site. Some content is difficult for older eyes to see. Guess I'll zoom in lol

Reply:
That's good feedback @brentschaus-okta . I know that the design team was aiming for a particular font size but I agree some of the diagrams are hard to read. Let's bring this up in the image guideline meeting.